dataland-keycloak/dataland_theme/email/html/template.ftl

```diff
@@ -0,0 +1,27 @@
+<#macro emailLayout>
+    <html>
+    <style>
+        p {
+            font-family: IBM Plex Sans, Helvetica;
+            font-weight: 500;
+            font-size: 16px;
+        }
+
+        a {
+            color: #e67f3f;
+        }
+    </style>
+    <head>
+        <link rel="Shortcut Icon" type="image/x-icon"
```

Comment:
Did you see an small x on which you could right click and then press download image?

Reply:
That downloaded the long dataland banner but not the one letter icon.

Reply:
In the design for the emails, only the long Dataland banner is supposed to be displayed in the mail.

The reason why I inserted this statement here into the "<head>" part is, because I read in the internet that if you do this, the Logo will be displayed as Favicon if you click "open mail in browser".
So I thought "why not", and inserted it here. If it works it works, if not, it doesn't break anything.